tests/e2e/unauthenticated_user_test/unathenticated_user_test.go

```diff
@@ -0,0 +1,84 @@
+// Unathenticated user tries to access endpoints
+
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/HackIllinois/api/common/configloader"
+	event_models "github.com/HackIllinois/api/services/event/models"
+	mail_models "github.com/HackIllinois/api/services/mail/models"
+	notification_models "github.com/HackIllinois/api/services/notifications/models"
+	"github.com/HackIllinois/api/tests/common"
+	"github.com/dghubble/sling"
+	"gopkg.in/mgo.v2"
+)
+
+var attendee_client *sling.Sling
+var session *mgo.Session
+
+func TestMain(m *testing.M) {
+
+	cfg, err := configloader.Load(os.Getenv("HI_CONFIG"))
+
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+
+	attendee_client = sling.New().Base("http://localhost:8000").Client(nil).Add("Authorization", "FAKE_TOKEN")
+
+	session = common.GetLocalMongoSession()
+
+	mail_db_name, err := cfg.Get("MAIL_DB_NAME")
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+	session.DB(mail_db_name).DropDatabase()
+
+	notification_db_name, err := cfg.Get("NOTIFICATIONS_DB_NAME")
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+	session.DB(notification_db_name).DropDatabase()
+
+	event_db_name, err := cfg.Get("EVENT_DB_NAME")
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+	session.DB(event_db_name).DropDatabase()
+
+	return_code := m.Run()
+	os.Exit(return_code)
+}
+
+func TestUnauthenticatedCalls(t *testing.T) {
+	// 1. Staff endpoint
+	received_mail_list := mail_models.MailList{}
+	response, _ := attendee_client.New().Get("/mail/list/").Receive(&received_mail_list, &received_mail_list)
+
+	if response.StatusCode != 403 {
+		t.Errorf("Unathenticated attendee able to access endpoint that requires authentication")
```

Comment:
Nit: typo Unauthenticated instead of Unathenticated. Also on lines 74, 82

Reply:
My bad. I'll fix this.